handler-bizkeeper/src/main/java/io/servicecomb/bizkeeper/BizkeeperHandler.java

```diff
@@ -52,6 +54,16 @@
         } catch (Exception e) {
             LOG.warn("Hystrix properties already registerd. Dynamic configuration may not work.");
         }
+        try {
+            HystrixPlugins.getInstance().registerCommandExecutionHook(new HystrixCommandExecutionHook() {
+                public <T> Exception onExecutionError(HystrixInvokable<T> commandInstance, Exception e) {
+                    LOG.warn("bizkeeper execution error", e);
+                    return e; //by default, just pass through
+                }
+            });
+        } catch (Exception e) {
```

Comment:
I don't think it's a good practice. Because IllegalStateException is thrown internally, we don't know what's it if we don't check the source code. This rule is acceptable when the exception is declared by the method implicitly and not a RuntimeException. When we deal with RuntimeException, we'd better to use generic exception type, It's better for code quality.